src/ducc0/fft/fft.h

```diff
@@ -174,10 +169,9 @@ struct util // hack to avoid duplicate symbols
     size_t parallel = size / (info.shape(axis) * vlen);
     if (info.shape(axis) < 1000)
       parallel /= 4;
-    size_t max_threads = ducc0::adjust_nthreads(nthreads);
```

Comment:
The `adjust_nthreads` call also interferes with the thread count, limiting it to 1 in the case of `DUCC0_NO_THREADING`, and to `DUCC0_NUM_THREADS` otherwise.  These limits are also imposed within `Distribution::exec*`, so by decoupling, the previous behavior is still maintained.

Reply:
OK, fine. It's a bit hairy, since I usually invoke `adjust_nthreads` as soon as possible, so that `nthreads` will reliably contain the actual number of threads afterwards. But I agree this is not necessary here, as far as I see.

BTW, if you think it makes things simpler conceptually, I'd be tiotally fine with having a dedicated macro like `DUCC0_TENSORFLOW_MODE`.

Reply:
Yes, that sounds good! If I see `DUCC0_NO_THREADING`, to my mind this implies "there is no multithreading going on", i.e. there is no need to worry about synchronization etc.